Doc/library/pathlib.rst

```diff
@@ -652,6 +652,18 @@ call fails (for example because the path doesn't exist):
    .. versionadded:: 3.5
 
 
+.. method:: Path.absolute()
+
+   Return an absolute version of this path based on the current working
+   directory.  This function works even if the path doesn't exist.
```

Comment:
This function => This method

I dislike "works even if", maybe: "This method doesn't check if the path exists." ?